src/Eviden.VirtualGrocer/Server/Program.cs

```diff
@@ -4,28 +4,28 @@
 using Microsoft.Identity.Web;
 using Eviden.VirtualGrocer.Web.Server.Skills.History;
 using System.Reflection;
+using Azure.Identity;
+
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Initialize the configuration
 var config = builder.Configuration;
 config.InitializeCommonConfiguration(Path.GetDirectoryName(Assembly.GetEntryAssembly()?.Location)!);
 
-// Register objects and services in the DI container
```

Comment:
I think we had these removed sections first and the config should go above it.

Reply:
Sounds good 👍🏼